src/applications/personalization/profile360/actions/index.js

```diff
@@ -35,9 +35,7 @@ export function fetchMilitaryInformation() {
     dispatch({
       type: FETCH_MILITARY_INFORMATION_SUCCESS,
       militaryInformation: {
-        serviceHistory: environment.isLocalhost()
```

Comment:
There is no need to mock this endpoint locally. This was only being done during initial development of the  Military Info component for Profile 2.0.